src/test/kube.rs

```diff
@@ -414,7 +414,10 @@ impl KubeClient {
         K: Clone + Debug + DeserializeOwned + Resource,
         <K as Resource>::DynamicType: Default,
     {
-        let get_value = |resource: &K| resource.meta().annotations.get(key).cloned();
+        let get_value = |resource: &K| match &resource.meta().annotations {
+            Some(annotations) => annotations.get(key).cloned(),
+            _ => None,
+        };
```

Comment:
This is okay. Personally, I find a chain more readable than a match:

```suggestion
        let get_value = |resource: &K| {
            resource
                .meta()
                .annotations
                .as_ref()
                .and_then(|annotations| annotations.get(key).cloned())
        };
```

Reply:
done